lib/Dialect/FIRRTL/Transforms/GrandCentral.cpp

```diff
@@ -2178,6 +2476,125 @@ void GrandCentralPass::runOnOperation() {
       continue;
   }
 
+  for (auto view : views) {
+    auto bundle = view.getAugmentedType();
+
+    assert(!bundle.isRoot() && "'id' found in firrtl.view");
+
+    auto viewParentMod = view->getParentOfType<FModuleLike>();
+    auto symbolName = getModuleNamespace(viewParentMod)
+                          .newName("__" + getInterfaceName(bundle) + "__");
+
+    // Recursively walk the AugmentedBundleType to generate interfaces and XMRs.
+    // Error out if this returns None (indicating that the view is
+    // malformed in some way).  A good error message is generated inside
+    // `traverseViewBundle` or the functions it calls.
```

Comment:
This comment is a bit far from the call to `traverseViewBundle`.

Looking now, this is in the original code. 🙃 Who wrote that?